common/src/main/java/com/thoughtworks/go/remote/work/artifact/ArtifactRequestProcessor.java

```diff
@@ -54,10 +53,10 @@ private GoApiResponse processConsoleLogRequest(GoPluginDescriptor pluginDescript
         final String message = format("[%s] %s", pluginDescriptor.id(), consoleLogMessage.getMessage());
         switch (consoleLogMessage.getLogLevel()) {
             case INFO:
-                goPublisher.taggedConsumeLine(PUBLISH, message);
+                goPublisher.taggedConsumeLine(TaggedStreamConsumer.OUT, message);
```

Comment:
The ArtifactRequestProcessor can be called from publish artifact as well. Make sure that the console log colour for publishing external artifacts remains consistent with publish of artifacts to gocd server.

Reply:
The difference is the two lines in white after `Start to upload pluggable artifact using...`. They were previously in blue. Is that acceptable? If not, we will probably have to change the contract of `ConsoleLogMessage`

Reply:
I think that having it in white is not consistent with the publish artifact to gocd server.